docs/Windows-DEVELOP.md

```diff
@@ -0,0 +1,112 @@
+# Windows Development Guide for rdiff-backup
+
+Some notes for developers and other people willing to help development on
+Windows platform, or simply compile rdiff-backup from source code on Windows.
+
+See also: https://wiki.python.org/moin/WindowsCompilers
+
+## Pre-requisites
+
+Here the list of required component to be install to start developing
+rdiff-backup on Windows platform.
+
+### Upgrade your Windows OS
+Don't overlook this step. Visual Studio required an upgraded OS.
+Use Windows Update to upgrade your system. Then proceed with the installation.
+
+### Visual Studio Build Tools
+
+1. To install Visual Studio Build Tools, you need to install .NET Framework 4.6.  
+   [Download .NET Framework 4.6](https://www.microsoft.com/en-US/download/details.aspx?id=53344)
+2. Finally, download Build Tools installation.  
+   [Microsoft Build Tools for Visual Studio 2019](https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2019)
+3. Once downloaded, the filename should be `vs_buildtools.exe` or similar, launch it to start the installation.
+4. From the user interface select "C++ Build Tools"
+5. In the right panel checked the following items. Everything else may be unchecked.

```

Comment:
Here I'm not sure and it might be your issue, as I installed finally from the command line. Perhaps you should run something like `"C:\Program Files (x86)\Microsoft Visual Studio\Installer\vs_installer.exe" modify --installPath "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools" --includeRecommended --norestart --quiet --add Microsoft.VisualStudio.Workload.VCTools` and see if it changes anything to the installation, then you'd know that you're missing something (I've used 2017, not 2019, so it might be slightly different).

Reply:
I which I could use VS2017. But since 2019 is out. It's impossible to get 2017 anymore.